### ISSUE TITLE
invalid_client invalid client error for spotify

### ISSUE THREAD

**Issue by RAWR8810** [codetheweb/muse#498]
Followed instruction for using docker. Spotify client id and secret have been checked and multiple ones have been tried.
![Screenshot 2022-02-01 230558](https://user-images.githubusercontent.com/94130108/152020583-13f34e9a-a96b-44e0-91d3-0d81762e842f.png)

**Comment by codetheweb** [comment]
Can you give more details on how you're running Muse?

**Comment by codetheweb** [comment]
Never mind, apparently that already exists.

I just changed the environment loading behavior slightly so it trims whitespace off of provided environment variables. That seems unlikely to be the issue but I can't think of anything else off the top of my head.

Try again with v0.5.3 and see what happens.

**Comment by RAWR8810** [comment]
Still no die
this is the code im executing
docker run -it -v "C:\Users\SANIC\Desktop\Utilities\DiscordBot\Muse bot\data":/data -e DISCORD_TOKEN='OTM4MDkyNDk0MzA0OTg1MTM4.YflQtw.B2HB5RyaYnMK2RsV_xo3A5IJvmY' -e SPOTIFY_CLIENT_ID='ce241d438b3b44cda18037f114774208' -e SPOTIFY_CLIENT_SECRET='53bb170053dc4d8192d7dcd95e90a8d4' -e YOUTUBE_API_KEY='AIzaSyB8s4FMi7UAf2-owLloAbx7ejKQqLfx_IYs' codetheweb/muse:0.5.4
![Screenshot 2022-02-02 131705](https://user-images.githubusercontent.com/94130108/152113680-43a53895-4545-41d3-ad84-1dd79ba358c9.png)

**Comment by RAWR8810** [comment]
same error on 0.5.3 as well
![Screenshot 2022-02-02 131943](https://user-images.githubusercontent.com/94130108/152114000-65acd0f6-1533-4a58-b1a5-7b4b8b57b107.png)

**Comment by RAWR8810** [comment]
@codetheweb please advise

**Comment by codetheweb** [comment]
I deleted the comment giving away your full credentials. I recommend redacting info in the future. 😉 

That being said, I used your credentials for my local instance of Muse and it worked just fine. The only difference between us at this point is OS and physical location.

I don't use Window's shell so I'm not exactly sure how quoting is handled. Maybe try passing in the environment variables as ` -e 'DISCORD_TOKEN=my-token'` (quotes on the outside) and see if that does anything?
